packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/start-job-run.test.ts

```diff
@@ -0,0 +1,1173 @@
+      Type: 'Task',
+      Resource: {
+        'Fn::Join': [
+          '',
+          [
+            'arn:',
+            {
+              Ref: 'AWS::Partition',
+            },
+            ':states:::emr-containers:startJobRun',
+          ],
+        ],
+      },
+      End: true,
+      Parameters: {
+        VirtualClusterId: clusterId,
+        ReleaseLabel: ReleaseLabel.EMR_6_2_0.label,
+        JobDriver: {
+          SparkSubmitJobDriver: {
+            EntryPoint: 'local:///usr/lib/spark/examples/src/main/python/pi.py',
+            SparkSubmitParameters: '--conf spark.executor.instances=2',
+          },
+        },
+        ConfigurationOverrides: {
+          MonitoringConfiguration: {
+            PersistentAppUI: 'ENABLED',
+          },
+        },
+        ExecutionRoleArn: {
+          'Fn::GetAtt': [
+            'EMRContainersStartJobRunJobExecutionRole40B8DD81',
+            'Arn',
+          ],
+        },
+      },
+    });
+  });
+
+  test('.sync integration pattern', () => {
+
+    // THEN
+    expect(stack.resolve(defaultTask.toStateJson())).toEqual({
+      Type: 'Task',
+      Resource: {
+        'Fn::Join': [
+          '',
+          [
+            'arn:',
+            {
+              Ref: 'AWS::Partition',
+            },
+            ':states:::emr-containers:startJobRun.sync',
+          ],
+        ],
+      },
+      End: true,
+      Parameters: {
+        VirtualClusterId: clusterId,
+        ReleaseLabel: ReleaseLabel.EMR_6_2_0.label,
+        JobDriver: {
+          SparkSubmitJobDriver: {
+            EntryPoint: 'local:///usr/lib/spark/examples/src/main/python/pi.py',
+            SparkSubmitParameters: '--conf spark.executor.instances=2',
+          },
+        },
+        ConfigurationOverrides: {
+          MonitoringConfiguration: {
+            PersistentAppUI: 'ENABLED',
+          },
+        },
+        ExecutionRoleArn: {
+          'Fn::GetAtt': [
+            'DefaultEMRContainersStartJobRunJobExecutionRole3A34E219',
+            'Arn',
+          ],
+        },
+      },
+    });
+  });
+
+  test('virtual cluster id from payload', () => {
+
+    // WHEN
+    const task = new EmrContainersStartJobRun(stack, 'EMR Containers Start Job Run', {
+      virtualCluster: VirtualClusterInput.fromTaskInput(sfn.TaskInput.fromJsonPathAt('$.ClusterId')),
+      releaseLabel: ReleaseLabel.EMR_6_2_0,
+      executionRole: iam.Role.fromRoleArn(stack, 'Job Execution Role', 'arn:aws:iam::xxxxxxxxxxxx:role/JobExecutionRole'),
+      jobDriver: {
+        sparkSubmitJobDriver: {
+          entryPoint: sfn.TaskInput.fromText('local:///usr/lib/spark/examples/src/main/python/pi.py'),
+          sparkSubmitParameters: '--conf spark.executor.instances=2',
+        },
+      },
+    });
+
+    // THEN
+    expect(stack.resolve(task.toStateJson())).toMatchObject({
+      Parameters: {
+        'VirtualClusterId.$': '$.ClusterId',
+        'ExecutionRoleArn': 'arn:aws:iam::xxxxxxxxxxxx:role/JobExecutionRole',
```

Comment:
```suggestion
        ExecutionRoleArn: 'arn:aws:iam::xxxxxxxxxxxx:role/JobExecutionRole',
```

Reply:
same as above, eslint, doesn't allow this to happen